jni/src/test/java/org/infinispan/client/jni/hotrod/JniTest.java

```diff
@@ -98,7 +98,10 @@ public static void main(String[] args) {
             "GetAllLocalTest.testBulkGetAfterLifespanExpire",
             "GetAllReplTest.testBulkGetAfterLifespanExpire",
             "RemoteCacheAdminTest.cacheReindexTest",                             // See HRCPP-469
-            "RemoteCacheAdminTest.cacheCreateWithEmbeddedConfigurationTest"      // conf to XML doesn't work here
+            "RemoteCacheAdminTest.cacheCreateWithEmbeddedConfigurationTest",     // conf to XML doesn't work here
+            "HotRodStatisticsTest.testHitsAndMisses",                            // We don't have client statistics here
+            "HotRodStatisticsTest.testStoresAndEntries",
```

Comment:
Any reason to skip the `testNumberOfEntriesAfterClear` and `testRemoveHitsAndMisses` in `HotRodStatisticsTest`?

Reply:
Those tests are executed, we're skipping (ignoring the failure actually) testHitsAndMisses and testStoresAndEntries because we don't have client statistics in c++ atm.